client/dwh_migration_client/macro_processor.py

```diff
@@ -273,6 +275,8 @@ def _get_all_regex_pattern_mapping(
         for file_map_key, token_map in macro_subst_maps.items():
             if fnmatch.fnmatch(file_path, file_map_key):
                 for key, value in token_map.items():
-                    reg_pattern_map[re.escape(key)] = value
+                    reg_pattern: AnyStr = "{0}{1}".format(self._case_insensitive_re_pattern, re.escape(key.lower())) \
+                        if use_reversed_map else re.escape(key)
+                    reg_pattern_map[reg_pattern] = value
         all_patterns = re.compile("|".join(reg_pattern_map.keys()))
```

Comment:
I get the following when I run `make test`:
```
dwh_migration_client/macro_processor.py:281: DeprecationWarning: Flags not at the start of the expression '(?i)my_table|(?i)mac' (truncated)
```
Is it possible to concat `(?i)` to the beginning of the compound pattern passed to `re.compile` instead each individual pattern?

Reply:
Yep, done.  Thanks.